core/client/fs/src/main/java/alluxio/client/file/RetryHandlingFileSystemMasterClient.java

```diff
@@ -232,7 +233,7 @@ public void mount(final AlluxioURI alluxioPath, final AlluxioURI ufsPath,
   @Override
   public void rename(final AlluxioURI src, final AlluxioURI dst)
```

Comment:
Does this one even have any callers? I don't see a reason to have an utility at client level. Could you double check and remove if this method is redundant?

Reply:
Removing would require changing the public-facing API. I don't think removing this is in-scope for the PR.

Reply:
This is not public API.

Reply:
Yes, but the public API calls this rename method